main.go

```diff
@@ -113,7 +112,6 @@ func main() {
 	registerComponentOrExit(mgr, operatorsv1alpha1.AddToScheme)
 	registerComponentOrExit(mgr, argoapi.AddToScheme)
 	registerComponentOrExit(mgr, configv1.AddToScheme)
-	registerComponentOrExit(mgr, consolepluginv1.AddToScheme)
```

Comment:
@jaideepr97 Thanks for the review. This line was removed purposely. Because after updating to v1, the console plugin API is the same as console API, see Line 40, 41. And the `console.AddToScheme` is already there in L 109
`registerComponentOrExit(mgr, console.AddToScheme)`

Reply:
@jaideepr97 Because when we started console plugin the plugin SDK feature was only on the console/v1alpha1 version, then later they updated it and the new feature in v1alpha1 got merged into the console/v1.